python/cudf/cudf/core/buffer/spillable_buffer.py

```diff
@@ -372,21 +369,8 @@ def __str__(self) -> str:
         )
 
 
-class SpillableBuffer(Buffer):
-    """A slice of a spillable buffer
-
-    This buffer applies the slicing and then delegates all
-    operations to its owning buffer.
-
-    Parameters
-    ----------
-    owner : SpillableBufferOwner
-        The owner of the view
-    offset : int
-        Memory offset into the owning buffer
-    size : int
-        Size of the view (in bytes)
-    """
+class SpillableBuffer(ExposureTrackedBuffer):
```

Comment:
Can we update the library design docs with the current state of things?

https://github.com/rapidsai/cudf/blob/branch-24.06/docs/cudf/source/developer_guide/library_design.md#copy-on-write

Reply:
I think it is still up to date